src/renderer/components/memory-renderer/PlotConfigurations.ts

```diff
@@ -0,0 +1,74 @@
+import { PlotData } from 'plotly.js';
+import { Chunk } from '../../../data/MemoryChunk';
+import { getBufferColor } from '../../../MemoryColorGenerator';
+import { formatSize, toHex } from '../../utils/numbers';
+
+export const L1RenderConfiguration = {
+    height: 110,
+    margin: {
+        l: 5,
+        r: 5,
+        b: 40,
+        t: 25,
+    },
+    title: 'L1 Address Space',
+} as PlotConfiguration;
```

Comment:
```suggestion
export const L1RenderConfiguration: PlotConfiguration = {
    height: 110,
    margin: {
        l: 5,
        r: 5,
        b: 40,
        t: 25,
    },
    title: 'L1 Address Space',
};
```

Nitpick: type casting can lead to hard to find bugs

Reply:
+1